frontend/app/module/polling_stations/PollingStationsLayout.tsx

```diff
@@ -0,0 +1,21 @@
+import { Outlet } from "react-router-dom";
+
+import { NavBar } from "app/component/navbar/NavBar";
+
+import { PollingStationListProvider } from "@kiesraad/api";
+import { PageTitle } from "@kiesraad/ui";
+import { useNumericParam } from "@kiesraad/util";
+
+export function PollingStationsLayout() {
+  const electionId = useNumericParam("electionId");
+
+  return (
+    <div className="app-layout">
+      <PageTitle title="Overzicht verkiezingen - Abacus" />
```

Comment:
The PageTitle here needs to be moved to PollingStationsListPage and the title renamed

Reply:
Moved to page in https://github.com/kiesraad/abacus/pull/453/commits/acb94c041a5dd7c7d9a4c24414812b98760791ec